mapit_gb/management/commands/mapit_UK_show_missing_codes.py

```diff
@@ -1,9 +1,24 @@
 from django.core.management.base import NoArgsCommand
 from mapit.models import Area, Generation
+from optparse import make_option
 
 
 class Command(NoArgsCommand):
     help = "Prints results of queries to check we aren't missing SNAC and GSS codes"
+    option_list = NoArgsCommand.option_list + (
+        make_option(
+            "--code_types", dest="code_types",
+            help="The list of code types to search for",
+            default="ons,gss"),
```

Comment:
I guess the real question is why do we look for missing codes on EUR areas - I'm not sure that we actually use those in GOV.UK do we?  Maybe we could just change that default.